test/message/source_map_throw_icu.js

```diff
@@ -0,0 +1,8 @@
+// Flags: --enable-source-maps
+
+'use strict';
+const hasICU = typeof Intl === 'object';
+if (hasICU) {
+  require('../common');
+  require('../fixtures/source-map/icu');
+}
```

Comment:
It should actually work without Intl as well - we have a fallback for that.

```suggestion
require('../common');
require('../fixtures/source-map/icu');
```